README.md

```diff
@@ -151,3 +151,78 @@ In particular, note:
 * [targets factory methods](https://records-mover.readthedocs.io/en/latest/records_mover.records.targets.html)
 * [move() method](https://records-mover.readthedocs.io/en/latest/records_mover.records.html#records_mover.records.move)
 * [BaseRecordsFormat](https://records-mover.readthedocs.io/en/latest/records_mover.records.html#records_mover.records.base_records_format.BaseRecordsFormat)
+
+## Local Development
+
+The included Dockerfile can be used to build a docker image that is
+suitable for local development.
+
+```bash
+docker build Dockerfile.dev -t records-mover:latest .
+```
+
+Or, using docker compose,
+```bash
+docker compose build
+```
+
+The following commands assume the records-mover container has been
+named `records-mover`.
+
+Mount the directory containing your local copy of the repository to
+have it override what's in the container.
+
+```bash
+docker run -it --mount src="$(pwd)/records_mover",target=/records-mover/records_mover,type=bind records-mover
```

Comment:
I don't know that we need the mounts anymore but tell me if I'm wrong.

Reply:
You need to mount your local files into the container to enable development inside an IDE. Otherwise it is unnecessary.